components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx

```diff
@@ -1,46 +1,99 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please revert changes in the license header

Reply:
changed with 910980c413d9aea940c47a751f949593c1fc5ec0